test/python/transpiler/test_gate_direction.py

```diff
@@ -65,9 +65,9 @@ def test_direction_error(self):
         """The mapping cannot be fixed by direction mapper
         qr0:---------
 
-        qr1:---(+)---
+        qr1:----.-----
                 |
-        qr2:----.----
+        qr2:---(+)----
```

Comment:
Is there a reason why these control and target qubits are being flipped in these tests?

Reply:
This is just updating the circuit drawings to match the actual test circuits in several docstrings.